test/gie/GDA.gie

```diff
@@ -50,6 +50,7 @@ operation proj = pipeline ellps=GRS80;
                z  = -0.04019;   rz = -0.0328979;  s = -0.009994;
           step proj = cart inv;
 -----------------------------------------------------------------------------------
+tolerance  1.1 mm
```

Comment:
That's an oddly specific tolerance. Is there a reason for that?

Reply:
Given the number of decimals given in the original publication you can hardly get closer, except by chance, but 2 mm seems a bit sloppy in a geodetic context. I will, however change to 2 mm for good (or rather: bad :-)) measure.